Update README.md to match new commands w/o -c (#103)
maximilien/i18n4go: README.md

### Getting Latest Executable: i18n4go
--------------------------------------

> Assuming you have a valid [Golang 1.19.5](https://golang.org/dl/) or [later](https://golang.org/dl/) installed for your system, you can quickly get the latest `i18n4go` executable by running the following `go` command:

Reviewer: Nit: I think we can lower the min go version. Maybe use 1.17? for the min?

Go modules came to effect as of 1.11 ([blog](https://go.dev/blog/migrating-to-go-modules))

```suggestion
Assuming you have a valid [Golang 1.17](https://golang.org/dl/) or [later](https://golang.org/dl/) installed for your system, you can quickly get the latest `i18n4go` executable by running the following `go` command:
```

```

> $ go get github.com/maximilien/i18n4go/i18n4go

Reviewer: Nit: Using `go get` is deprecated for installing binaries.  It is recommend by google to use `go install` to install binaries and compiled packages ([doc](https://pkg.go.dev/cmd/go#hdr-Compile_and_install_packages_and_dependencies))

Also, as of 1.18 `go get` will not build packages so it is better to use `go install` in the long run. ([blog](https://go.dev/doc/go-get-install-deprecation))

```suggestion
$ go install github.com/maximilien/i18n4go/i18n4go
```

> Printing the usage help: `$ i18n4go -h` or `$ i18n4go --help`

Reviewer: Typing `i18n4go` prints out the help text too.

```suggestion
Printing the usage help: `$ i18n4go` . 
Printing  usage help for commands: `$ i18n4go <command> --help|-h`
